continuous_integration/install.sh

```diff
@@ -76,7 +76,7 @@ if [ -z "$NUMPY_VERSION" ]; then
 fi
 
 if [[ "$COVERAGE" == "true" ]]; then
-    pip install coverage coveralls
+    pip install coverage coveralls memory_profiler
```

Comment:
You should probably experiment with memory_profiler-dependent tests in a separate branch